src/components/four-up.tsx

```diff
@@ -63,18 +66,27 @@ export class FourUpComponent extends BaseComponent<IProps, IState> {
   }
 
   public componentDidMount() {
-    if (this.container) {
-      this.grid.update({
-        height: this.container.offsetHeight - BORDER_SIZE,
-        initSplitters: true,
-        width: this.container.offsetWidth,
-      });
-    }
-    window.addEventListener("resize", this.handleResizeWindow);
```

Comment:
We should be able to remove the entire `handleResizeWindow` function now as well.